src/mono/wasm/debugger/DebuggerTestSuite/Inspector.cs

```diff
@@ -156,7 +156,7 @@ void FailAllWaiters(Exception? exception = null)
             }
         }
 
-        private (string line, string type) FormatConsoleAPICalled(JObject args)
+        internal (string line, string type) FormatConsoleAPICalled(JObject args)
```

Comment:
Is this needed?

Reply:
This is being called by `DebuggerTestBase` now.